tests/lab01_test.go

```diff
@@ -29,10 +29,14 @@ var _ = Describe("Lab 1", func() {
 		})
 
 		It("should have a valid skaffold.yaml", func() {
-			skaffoldExpected := ExpectYamlToParse("../skaffold.yaml")
+			skaffoldExpected, errorMessage := ExpectYamlToParse("../skaffold.yaml")
+			if errorMessage != "" {
+				failMessage = errorMessage
+			}
```

Comment:
Couldn't you just set failMessage directly on line 32?

Reply:
So when I tried to directly pass the failMessage, it wouldn't pass the value of failMessage down to the AfterEach to concatenate the value into the `slack-output.md`. I didn't understand why exactly the value isn't passed but this was the workaround for that